journal/modernized/2-journal.adoc

```diff
@@ -5857,6 +5859,9 @@ which had a very great effect upon the meeting,
 and it was concluded to make a minute expressive of the feelings produced,
 and of its sympathy with me, and desires for my support; it being as follows:
 
+[.embedded-content-document.letter]
```

Comment:
Hmmm... I think we should add another _type_ -- it will be pretty rare, but useful I think.  Let's change this to `.minute`.